frontend/pages/projects/_id/affective-annotation/index.vue

```diff
@@ -89,6 +89,7 @@
                 />
               </div>
             </v-card>
+            <component :is="affectiveAnnotationComponent" v-if="affectiveAnnotationComponent" />
           </v-col>
```

Comment:
I have seen your code thoroughly, and with your approach, yes; it will be kind of hard to integrate them . The props can be set like usual actually, but it will not be clean and readable so I will discard this approach and follow yours.  Better than rewriting the whole code. XD